concourse/pipelines/create-deployer.yml

```diff
@@ -514,6 +519,29 @@ jobs:
         params:
           file: concourse-deploy/concourse-manifest-state.json
 
+    - task: add-env-specific-team
+      config:
+        image: docker:///governmentpaas/self-update-pipelines
+        inputs:
+          - name: paas-cf
+          - name: concourse-manifest
+        params:
+          SYSTEM_DNS_ZONE_NAME: {{system_dns_zone_name}}
+          CONCOURSE_ATC_PASSWORD: {{concourse_atc_password}}
+          CONCOURSE_ATC_USER: admin
+          FLY_TARGET: {{deploy_env}}
+          FLY_CMD: ./paas-cf/bin/fly
+        run:
+          path: sh
+          args:
+            - -e
+            - -c
+            - |
+              export CONCOURSE_URL="https://deployer.${SYSTEM_DNS_ZONE_NAME}"
+              ./paas-cf/concourse/scripts/fly_sync_and_login.sh
+              echo y | ./paas-cf/bin/fly -t {{deploy_env}} set-team -n {{aws_account}} \
```

Comment:
I could export FLY_TARGET in the script directly. This is for the login script and it has to be exported. Doing it as param is simpler (less code, less questioning). Passing team name and aws account as params adds 2 params which I only use once. Instead I can save 2 lines of code and use fly variables in the code directly...